ethportal-api/src/types/execution/header.rs

```diff
@@ -557,6 +605,109 @@ mod tests {
+    // Test vector from: https://github.com/ethereum/tests/blob/7e9e0940c0fcdbead8af3078ede70f969109bd85/BlockchainTests/ValidBlocks/bcExample/cancunExample.json
+    #[test]
+    fn dencun_rlp_ethereum_tests_example() {
+        let data = hex_decode("0xf90221a03a9b485972e7353edd9152712492f0c58d89ef80623686b6bf947a4a6dce6cb6a01dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347942adc25665018aa1fe0e6bc666dac8fc2697ff9baa03c837fc158e3e93eafcaf2e658a02f5d8f99abc9f1c4c66cdea96c0ca26406aea04409cc4b699384ba5f8248d92b784713610c5ff9c1de51e9239da0dac76de9cea046cab26abf1047b5b119ecc2dda1296b071766c8b1307e1381fcecc90d513d86b90100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008001887fffffffffffffff8302a86582079e42a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b42188000000000000000009a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b4218302000080").unwrap();
+        let decoded: Header = rlp::decode(&data).unwrap();
+        let expected: Header = Header {
+            parent_hash: H256::from_str(
+                "3a9b485972e7353edd9152712492f0c58d89ef80623686b6bf947a4a6dce6cb6",
+            )
+            .unwrap(),
+            uncles_hash: H256::from_str(
+                "1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+            )
+            .unwrap(),
+            author: H160::from_str("2adc25665018aa1fe0e6bc666dac8fc2697ff9ba").unwrap(),
+            state_root: H256::from_str(
+                "3c837fc158e3e93eafcaf2e658a02f5d8f99abc9f1c4c66cdea96c0ca26406ae",
+            )
+            .unwrap(),
+            transactions_root: H256::from_str(
+                "4409cc4b699384ba5f8248d92b784713610c5ff9c1de51e9239da0dac76de9ce",
+            )
+            .unwrap(),
+            receipts_root: H256::from_str(
+                "46cab26abf1047b5b119ecc2dda1296b071766c8b1307e1381fcecc90d513d86",
+            )
+            .unwrap(),
+            logs_bloom: Bloom::default(),
+            difficulty: U256::from_str("0").unwrap(),
+            number: 0x1,
+            gas_limit: U256::from_str("0x7fffffffffffffff").unwrap(),
+            gas_used: U256::from_str("0x02a865").unwrap(),
+            timestamp: 0x079e,
+            extra_data: vec![0x42],
+            mix_hash: Some(
+                H256::from_str("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")
+                    .unwrap(),
+            ),
+            nonce: Some(H64::zero()),
+            base_fee_per_gas: Some(U256::from_str("9").unwrap()),
```

Comment:
Why are some of these `0x` prefixed (eg. `gas_used`) and some not? I think we should be consistent